src/services/chris_integration.tsx

```diff
@@ -108,48 +122,21 @@ class ChrisIntegration {
     return count;
   }
 
-  // old code used for manually uploaded image analysis
-  static async processNewAnalysis(files: LocalFile[]): Promise<boolean> {
+  static async processOneImg(img: DcmImage, xrayModel: string, ctModel: string): Promise<BackendPollResult[]> {
     let client: any = await ChrisAPIClient.getClient();
-    try {
-      // upload file
-      const uploadedFile = await client.uploadFile({
-        "upload_path": `chris/uploads/covidnet/${files[0].name}`
-      }, { "fname": files[0].blob })
-
-      // create dircopy plugin
-      const dircopyPlugin = (await client.getPlugins({ 'name_exact': this.FS_PLUGIN })).getItems()[0];
-      const data: DirCreateData = { "dir": uploadedFile.data.fname }
-      const pluginInstance: PluginInstance = await client.createPluginInstance(dircopyPlugin.data.id, data);
+    
+    let XRayModels: { [id: string]: string } = app.XrayModels; // Destructuring
+    let CTModels: { [id: string]: string } = app.CTModels;
 
-      await pollingBackend(pluginInstance)
+    let XRayModel: string = XRayModels[xrayModel]; // Configuring ChRIS to use the correct Xray model
+    let CTModel: string = CTModels[ctModel]; // Configuring ChRIS to use the correct CT model
```

Comment:
Are `XRayModels` and `CTModels` one time use variables? If so, could we just directly get the values from `app.XrayModels?.[xrayModel]`?

Nitpicking, if you could think of another name for those variable, it would be great because both `XRayModel` and `xrayModel` are the same words with different cases and are both strings so it might be easy to mix them up.

Reply:
Unfortunately, `app.XrayModels?.[xrayModel]` still gives the typing error. I've tried a few other ways as well, and none seem to work. The current version is the only working solution that we have

Reply:
Have you tried adding an interface?